### ISSUE TITLE
Sidekiq endpoint protected with Devise has a redirection loop

### ISSUE THREAD

**Issue by gottfrois** [mperham/sidekiq#2542]
In my routes:

```ruby
authenticate :user, ->(user) { user.admin? } do
  mount Sidekiq::Web => '/sidekiq'
end
```

If I hit `http://mysite.com/sidekiq` while not logged in, it redirects me to `http://mysite.com/sidekiq/users/sign_in`

Am I missing something?

**Comment by ryansch** [comment]
This is a devise bug. Upgrade to the latest version.

**Comment by gottfrois** [comment]
Right, thanks for the hint !

> Fix infinite redirect in Rails 4.2 authenticated routes

https://github.com/plataformatec/devise/blob/v3.5.2/CHANGELOG.md
